Web Inspector: REGRESSION(301917@main) Console messages are not rendered

rcaliman-apple opened the pull request:
#### ade3d3fcdde501b4154f10a922582471ce452abf
<pre>
Web Inspector: REGRESSION(301917@main) Console messages are not rendered
<a href="https://bugs.webkit.org/show_bug.cgi?id=301339">https://bugs.webkit.org/show_bug.cgi?id=301339</a>
<a href="https://rdar.apple.com/163256970">rdar://163256970</a>

Reviewed by Devin Rousso.

Changes in <a href="https://bugs.webkit.org/show_bug.cgi?id=300728">https://bugs.webkit.org/show_bug.cgi?id=300728</a> removed checks to ensure the message view
to be appended is an instance of `WI.ConsoleMessageView`. It can also be a `WI.ConsoleCommandView`.

* Source/WebInspectorUI/UserInterface/Controllers/JavaScriptLogViewController.js:
(WI.JavaScriptLogViewController.prototype._appendConsoleMessageView):
(WI.JavaScriptLogViewController.prototype._appendConsoleCommandView):
(WI.JavaScriptLogViewController.prototype._addToPendingMessages):

Canonical link: <a href="https://commits.webkit.org/302032@main">https://commits.webkit.org/302032@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/82f3652993e2410cbdcb9f6c04978a858df4d26b

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/127773 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/47421 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/38591 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/135114 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/79330 "Built successfully") | [✅ 🛠 ios-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/f2c6d2ed-bd8b-4d70-a499-64172ae81bd0/3bae5acf-c02d-4e8f-b281-f5fa23d561bf) 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/48051 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/55952 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/97254 "Passed tests") | [❌ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/65162 "") | [✅ 🛠 mac-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/f2c6d2ed-bd8b-4d70-a499-64172ae81bd0/f5a1843a-0e00-4730-9762-5ec4db5bd40a) 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/130721 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/38421 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/114453 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/77736 "Passed tests") | | [✅ 🛠 vision-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/f2c6d2ed-bd8b-4d70-a499-64172ae81bd0/85ffbed5-95f0-4a2f-b331-762e79b96fce) 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/37214 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/32554 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/78455 "Built successfully") | | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/108300 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/33012 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/137572 "Built successfully") | | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/54432 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/41968 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/105790 "Passed tests") | | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/54943 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/110808 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/105451 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/26899 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/50954 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/29388 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/52063 "Built successfully") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/54370 "Built successfully") | [✅ 🛠 mac-safer-cpp](https://ews-build.webkit.org/#/builders/120/builds/60718 "Built successfully") | [⏳ 🛠 jsc-armv7 ](https://ews-build.webkit.org/#/builders/JSC-ARMv7-32bits-Build-EWS "Waiting in queue, processing has not started yet") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/53606 "Built successfully") | | [⏳ 🧪 jsc-armv7-tests ](https://ews-build.webkit.org/#/builders/JSC-ARMv7-32bits-Build-EWS "Waiting in queue, processing has not started yet") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/57060 "Built successfully") | | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/55362 "Built successfully") | | | | 
<!--EWS-Status-Bubble-End-->

dcrousso reviewed (approved):
oh lol oops.  my mistake for assuming that `WI.ConsoleCommandView` inherited from `WI.ConsoleMessageView`

rcaliman-apple committed:
Web Inspector: REGRESSION(301917@main) Console messages are not rendered
https://bugs.webkit.org/show_bug.cgi?id=301339
rdar://163256970

Reviewed by Devin Rousso.

Changes in https://bugs.webkit.org/show_bug.cgi?id=300728 removed checks to ensure the message view
to be appended is an instance of `WI.ConsoleMessageView`. It can also be a `WI.ConsoleCommandView`.

* Source/WebInspectorUI/UserInterface/Controllers/JavaScriptLogViewController.js:
(WI.JavaScriptLogViewController.prototype._appendConsoleMessageView):
(WI.JavaScriptLogViewController.prototype._appendConsoleCommandView):
(WI.JavaScriptLogViewController.prototype._addToPendingMessages):

Canonical link: https://commits.webkit.org/302032@main
